### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by cokelaer**
Update README documentation of various wrappers (e.g; bwa, minimap2, etc)

### CODE REVIEW

File: wrappers/add_read_group/README.md [hunk 1]
```diff
@@ -1,31 +1,46 @@
 # Documentation
 
-This wrapper add a read group in an input BAM file. 
-5 standard read tags are handle by the wrapper but any other can be added using
+addis a read group in an input BAM file.
```
Comment (khourhin) [inline wrappers/add_read_group/README.md:3]: addis > This wrapper add ?

File: wrappers/add_read_group/README.md [hunk 2]
```diff
@@ -1,31 +1,46 @@
...
-Log:
+**Log:**
 
 - a log file with stdout/stderr
 
+**Notes:**
+
+If not provided, PL is set to 'Illumina', PU is set to 'unknown' and
+LB is set to 'unknown'. If not provided, SM is set to the input BAM file
+filename (without .bam extension). You can also use a wildcards.
```
Comment (khourhin) [inline wrappers/add_read_group/README.md:28]: a wildcards > wildcards

File: wrappers/add_read_group/README.md [hunk 3]
```diff
@@ -1,31 +1,46 @@
...
-    snpeff:  
-        annotation_file:  # the genbank file
+    add_read_group:
         options:    # result filters options
+        # PL:   (automatically filled with Illumina)
+        # PU:   (automatically filled with unknown)
+        # LB:   (automatically filled with unknown)
+        # SM:   (automatically filled with sample name from 
+                 input file. e.g. file.sorted.bam returns 'file')
+        # ID    (automaticall set to a unique uuid)
```
Comment (khourhin) [inline wrappers/add_read_group/README.md:42]: automaticall > automatically

File: wrappers/bowtie1/build/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+# Documentation
+
+builcs a genome index with Bowtie2
```
Comment (khourhin) [inline wrappers/bowtie1/build/README.md:3]: builcs > builds

File: wrappers/bowtie1/build/README.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+This wrapper takes a reference and build its index. The output files are
```
Comment (khourhin) [inline wrappers/bowtie1/build/README.md:5]: build > builds

File: wrappers/bowtie2/build/README.md [hunk 6]
```diff
@@ -1,35 +1,35 @@
 # Documentation
 
-Genome indexing with Bowtie2
+builcs a genome index with Bowtie2
```
Comment (khourhin) [inline wrappers/bowtie2/build/README.md:3]: builcs > builds

File: wrappers/bowtie2/build/README.md [hunk 7]
```diff
@@ -1,35 +1,35 @@
...
 
-This wrapper takes the reference and build its index. The output files are 
-stored in a directory extracted from the first output filename.
+This wrapper takes a reference and build its index. The output files are
```
Comment (khourhin) [inline wrappers/bowtie2/build/README.md:5]: build > builds

File: wrappers/multiqc/README.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+- list of files expected by MultiQC
+
+**Required output:**
+
+- **done**: a filename used as a trigger of job done. Be aware that the root
+  directory of this file is used to store the results. 
+
+**Required parameters:**
+
+- **options**: a list of valid FastQC options
```
Comment (khourhin) [inline wrappers/multiqc/README.md:16]: FastQC > MultiQC

File: wrappers/multiqc/README.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+- **working_directory**: place where results of FastQC are saved
```
Comment (khourhin) [inline wrappers/multiqc/README.md:17]: FastQC > MultiQC

File: wrappers/multiqc/README.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+**Log:**
+
+- a log file generated by FastQC is stored
```
Comment (khourhin) [inline wrappers/multiqc/README.md:21]: FastQC > MultiQC

File: wrappers/multiqc/README.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+
+# Configuration
+
+    ##############################################################################
+    # FastQC section
```
Comment (khourhin) [inline wrappers/multiqc/README.md:26]: FastQC > MultiQC

File: wrappers/multiqc/README.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+    #
+    # :Parameters:
+    #
+    # - options: string with any valid FastQC options
```
Comment (khourhin) [inline wrappers/multiqc/README.md:30]: FastQC > MultiQC

File: wrappers/multiqc/README.md [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+    fastqc:
```
Comment (khourhin) [inline wrappers/multiqc/README.md:31]: fastqc > multiqc

**Review by khourhin — APPROVED** [review]

File: wrappers/bwa/align/README.md [hunk 14]
```diff
@@ -1,47 +1,62 @@
...
 # Configuration
 
+    #############################################################################
+    #  Bowtie2 read mapping
+    #
+    # :Parameters:
+    #
+    # - options: any options recognised by 'bowtie2 index' command
+    # - threads: number of threads to be used
+    bowtie2:
```
Comment (ddesvillechabrol) [inline wrappers/bwa/align/README.md:40]: C'est pas bwa normalement ici ?

File: wrappers/minimap2/README.md [hunk 15]
```diff
@@ -0,0 +1,55 @@
...
+- **options**: a list of valid fastqc options
+
+**Log:**
+
+- a log file generated by bowtie2 is stored
+
+# Configuration
+
+    #############################################################################
+    #  Bowtie2 read mapping
```
Comment (ddesvillechabrol) [inline wrappers/minimap2/README.md:28]: minimap2 read mapping

**Review by ddesvillechabrol — APPROVED** [review]

**Pull request merged**